httpserver/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ loki_add_subdirectory(../pow pow)
 loki_add_subdirectory(../crypto crypto)
 find_package(OpenSSL REQUIRED)
 
+set(Boost_USE_STATIC_LIBS ON)
```

Comment:
Have you tried overriding it on the command line?
I wonder if we should do `if (NOT DEFINED ...)` to ensure we allow the user to override it...

Reply:
I was unable to even *define* it from the command line, I would have been happy without it otherwise
Not sure if this is just something I'm messing up?

Reply:
'Twas